harness/determined/cli/command.py

```diff
@@ -130,20 +131,22 @@
 }
 
 
-Command = namedtuple(
-    "Command",
-    [
-        "id",
-        "owner",
-        "registered_time",
-        "config",
-        "state",
-        "addresses",
-        "exit_status",
-        "misc",
-        "agent_user_group",
-    ],
-)
+ls_sort_args: declarative_argparse.ArgsDescription = [
```

Comment:
There's a https://github.com/determined-ai/determined/blob/main/harness/determined/cli/top_arg_descriptions.py. What do you think about put this there?

And, yuck, _also_ precedence for putting shared flags in https://github.com/determined-ai/determined/blob/main/harness/determined/cli/_util.py#L13.

I don't like any solution here. Inclination is to choose an existing suboptimal "standard" rather than create a new one. But if those other choices are suboptimal enough, I think this is fine.

Reply:
I think the shared flags case has the flags which are used for a bunch of different subcommands all over the code, and these two flags are really specific for `command.list_tasks`. so i've put it next to `list_tasks`.

Reply:
cool, cool. This choice works for me.